internal/db/migrations/002_utility_functions.sql

```diff
@@ -41,13 +41,13 @@ BEGIN
 END;
 $provider_type_azure$ LANGUAGE 'plpgsql' IMMUTABLE PARALLEL SAFE;
 
-CREATE OR REPLACE FUNCTION provider_type_gcp()
+CREATE OR REPLACE FUNCTION provider_type_gce()
   RETURNS INTEGER AS
-$provider_type_gcp$
+$provider_type_gce$
 BEGIN
   RETURN(SELECT 4);
 END;
-$provider_type_gcp$ LANGUAGE 'plpgsql' IMMUTABLE PARALLEL SAFE;
+$provider_type_gce$ LANGUAGE 'plpgsql' IMMUTABLE PARALLEL SAFE;
```

Comment:
You cannot do this, modifications of existing SQL files are no longer allowed. You have to do ALTER FUNCTION RENAME TO kind of DDL statement.